pkg/kv/kvserver/queue_test.go

```diff
@@ -690,24 +690,6 @@ func TestAcceptsUnsplitRanges(t *testing.T) {
 	bq := makeTestBaseQueue("test", testQueue, s, queueConfig{maxSize: 2})
 	bq.Start(stopper)
 
-	// Check our config.
-	var sysCfg *config.SystemConfig
-	testutils.SucceedsSoon(t, func() error {
-		sysCfg = s.cfg.Gossip.DeprecatedGetSystemConfig()
-		if sysCfg == nil {
-			return errors.New("system config not yet present")
-		}
-		return nil
-	})
-	neverSplitsDesc := neverSplits.Desc()
-	if sysCfg.NeedsSplit(ctx, neverSplitsDesc.StartKey, neverSplitsDesc.EndKey) {
```

Comment:
Down in KV we have an alternative for the component that determines whether a keyspan has hard split points between them, you should be be able to get yourself a spanconfig.StoreReader (GetConfReader?) backed by the span configs infra, and have this test use that instead. Ditto below; this test is a bit hollowed out without it.

Reply:
Done